### ISSUE TITLE
[ETHEREUM-CONTRACT] [GDA] Deploy GDA featuresets to (almost) all testnets

### ISSUE THREAD

**Issue by hellwolf** [superfluid-finance/protocol-monorepo#1601]
In order to limit as much as surprises before the GDA rollout, we should have GDA feature on all testnets as early as possible. That should include contracts, subgraph endpints, superfluid user interfaces and solvency network that supports GDA.

- Testnet Support Matrix:
  - base-goerli, polygon-zkevm-testnet, eth-sepolia: in sync with mainnet featureset.
  - Rest of the network will be (after this issue being clsoed) shipped with GDA feature set.

# TODOs

- [x] (BLOCKER) Many super token factories on testnets are currently kaput. It is pending to be fixed in https://github.com/superfluid-finance/superfluid-stories/issues/378
- [x] Deploy GDA branch to all gda-enabled testnets
- [x] Publish subgraph endpoints to thegraph-hosted service from GDA branch.
  - [x] deplopyment script improvement: https://github.com/superfluid-finance/protocol-monorepo/pull/1624
  - [x] eth-goerli https://thegraph.com/hosted-service/subgraph/superfluid-finance/protocol-v1-goerli
  - [x] ~eth-sepolia (to be self-hosted)~ non-GDA
  - [x] arbitrum-goerli https://thegraph.com/hosted-service/subgraph/superfluid-finance/protocol-v1-arbitrum-goerli
  - [x] avalanche-fuji https://thegraph.com/hosted-service/subgraph/superfluid-finance/protocol-v1-avalanche-fuji
  - [x] optimism-goerli https://thegraph.com/hosted-service/subgraph/superfluid-finance/protocol-v1-optimism-goerli
  - [x] ~polygon-zkevm-testnet (to be self-hosted, or what ???)~ non-GDA
  - [x] polygon-mumbai https://thegraph.com/hosted-service/subgraph/superfluid-finance/protocol-v1-mumbai
  - [x] ~base-goerli (to be self-hosted)~ non-GDA
- [x] Setup sentinel network with GDA branch support.
  - Sentinel support: https://github.com/superfluid-finance/superfluid-sentinel/pull/239
  - ...

**Comment by d10r** [comment]
deployments: https://github.com/superfluid-finance/superfluid-stories/issues/378#issuecomment-1686924482

**Comment by d10r** [comment]
## subgraph deployment:
optimism-goerli failed: https://thegraph.com/hosted-service/subgraph/superfluid-finance/protocol-v1-optimism-goerli (retrying after a fix from Vincent)
arbitrum goerli synced: https://thegraph.com/hosted-service/subgraph/superfluid-finance/protocol-v1-arbitrum-goerli
avalanche-fuji synced: https://thegraph.com/hosted-service/subgraph/superfluid-finance/protocol-v1-avalanche-fuji
polygon-mumbai synced: https://thegraph.com/hosted-service/subgraph/superfluid-finance/protocol-v1-mumbai
eth-goerli: syncing: https://thegraph.com/hosted-service/subgraph/superfluid-finance/protocol-v1-goerli

Working on simplification of the workflow code, too much redundancy.

**Comment by d10r** [comment]
gda-enabled sentinels started for eth-goerli, optimism-goerli and polygon-mumbai.
Running on a new VM solvency-staging.x.superfluid.dev
Can't yet access monitoring ports bcs. blocked by GCP default firewall.

**Comment by hellwolf** [comment]
Still no easy observability and ways to tell if they are actuall working. Deferring acceptance testing to the separate ticket related to https://github.com/superfluid-finance/superfluid-sentinel/issues/237
